lectures/inflation_history.md

```diff
@@ -360,63 +386,66 @@ for i in range(4):
 df_Aus, df_Hung, df_Pol, df_Germ = df_list
 ```
 
-Let's  construct graphs for our four countries.
+Now let's construct graphs for our four countries.
 
 For each country, we'll plot two graphs.
 
 The first graph plots logarithms of 
 
-  * price levels
-  * exchange rates vis a vis US dollars
+* price levels
+* exchange rates vis a vis US dollars
 
-For each country, the scale on the right side of a graph will pertain to the price level while the scale on the left side of a graph will pertain
-to the exchange rate. 
+For each country, the scale on the right side of a graph will pertain to the price level while the scale on the left side of a graph will pertain to the exchange rate. 
 
-For each country, the second graph plots a three-month moving average of the inflation rate defined as $p_t - p_{t-1}$.
+For each country, the second graph plots a centered three-month moving average of the inflation rate defined as $\frac{p_{t-1} + p_t + p_{t+1}}{3}$.
```

Comment:
Is this a moving average of prices or of the inflation rate?

Reply:
`inflation` rate